### ISSUE TITLE
Lightning control

### ISSUE THREAD

**Issue by flutschepfeil** [banban525/echonetlite2mqtt#16]
Hello,
First let me thank you for the amazing project, it seems like just the perfect solution for me. Also, let me apologize, because this is not exactly an issue, but a question.
I'm using home assistant and want to make my light switches smart. So far I only found the Panasonic Advance (Plus) series.
Is my assumption correct that I need the Link-plus wireless adapter to read out the states and issue commands?

I was hoping that you can give me a small push in the right direction and how to start off.

**Comment by aftertommy** [comment]
Unfortunately link plus itself isn’t echonetlite compatible. The Aiseg2 adapter makes them controllable via the web browser, and it also allows (limited) control of echonetlite devices via web browser, but it doesn’t do any bridging to echonetlite. 

There exists a limited homebridge project for controlling link plus lights, but none for home assistant. The work involved appears straightforward— it’s a matter of building a scraper. But someone just has to do it.

**Comment by aftertommy** [comment]
Here’s the homebridge project:

https://github.com/vwhitteron/homebridge-aiseg2

**Comment by banban525** [comment]
@flutschepfeil 

I have no connection with Panasonic, so I can't give you an exact answer.

ECHONETLite2MQTT is an application that communicates using the ECHONET Lite specifications.
We can check supported devices on the ECHONET website.

I found a wireless adapter for the Panasonic Link Plus Switch.

* ja: https://echonet.jp/introduce/gz-000755/
* en: https://echonet.jp/introduce_en/gz-000755/

If your switch is Panasonic Link Plus, you may be able to control it with ECHONETLite2MQTT using this device.

However, I do not recommend trying it, because this device is expensive and you may not be able to control the switch.

**Comment by aftertommy** [comment]
Yeah you won’t be able to control the switches with that adapter. 

That said, if you’re handy with coding you could build a bridge that allows you to control the lights over MQTT (or whatever).

**Comment by flutschepfeil** [comment]
Apologies for the late reply. Thanks for all the ideas and answers. I have sadly no experience with coding, so creating something from scratch will be difficult for me.

As for the homebridge, I would need an AppleTV to make use if it I assume?

Are there in general smart switches (tactile, also 3way or 4way) that would work with home assistant in Japan?
@banban525 may I ask if/what you are currently using as smart switches? :)

**Comment by aftertommy** [comment]
Can confirm! WTY2001 works great.

**Comment by flutschepfeil** [comment]
@aftertommy and mathewbadeau many man thanks for the information and link to the other thread, this look like amazing news! :)
